modules/cnf-use-device-interrupt-processing-for-isolated-cpus.adoc

```diff
@@ -0,0 +1,24 @@
+// CNF-802 Infrastructure-provided interrupt processing for guaranteed pod CPUs
+// Module included in the following assemblies:
+//
+// *cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+[id="use-device-interrupt-processing-for-isolated-cpus_{context}"]
+= Upgrading the performance profile to use device interrupt processing for isolated CPUs
```

Comment:
Here again,  ""for isolated CPUs" is not rally about this, is about be able to load balance device interrupts across ALL CPUs as long as they don't belong to a guaranteed pod